pkg/controller/direct/apikeys/apikeyskey_controller.go

```diff
@@ -284,7 +284,27 @@ func (a *adapter) Update(ctx context.Context, u *unstructured.Unstructured) erro
 }
 
 func (a *adapter) Export(ctx context.Context) (*unstructured.Unstructured, error) {
-	return nil, nil
+	if a.actual == nil {
+		return nil, fmt.Errorf("apikeyskey %q not found", a.fullyQualifiedName())
+	}
+
+	spec := direct.LazyPtr(a.actual.Spec)
+	specObj, err := runtime.DefaultUnstructuredConverter.ToUnstructured(spec)
```

Comment:
We might want to move apikeyskey to our generated mappers (and then I think we can get rid of the mapping layer entirely)...

... but for the moment I guess this is a good step forwards!